src/plugins/intel_gna/gna_graph_compiler.cpp

```diff
@@ -1242,7 +1242,9 @@ void GNAGraphCompiler::EltwisePrimitive(InferenceEngine::CNNLayerPtr layer) {
     }
 
     if (in_4b_total_size != in_2b_total_size) {
-        THROW_GNA_LAYER_EXCEPTION(layer) << " Inputs size mismatch " << in_4b_total_size << " != " << in_2b_total_size;
+        THROW_GNA_LAYER_EXCEPTION(layer) << " Inputs size mismatch "
+            << "(note: For Multiply, Add and Subtract layers auto broadcasting is only supported for inputs which are constant) "
```

Comment:
@andrew-zaytsev suggests to replace with "constant inputs".

Reply:
done